openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -79,6 +84,7 @@ The following items are out of scope for the current version of this document, b
 
 * Trust Management, i.e. authorization of an issuer to issue certain types of credentials, authorization of the Wallet to be issued certain types of credentials, authorization of  the Verifier to receive certain types of credentials.
 * Protocol for presentation of Verifiable Credentials for offline use-cases, e.g. over BLE.
+* Profile of OID4VP without using W3C Digital Credentials API to present ISO mdocs is defined in [@!ISO.18013-7]. For more details, also see Annex B.3 in [@!OIDF.OID4VP].
```

Comment:
ISO/IEC TS 18013-7:2024 is using OID4VP ID-2. HAIP would be incompatible with 18013-7 the moment it gets published which is why it is problematic to just refer to 18013-7. Instead, we should define a URI scheme version of OID4VP for mdocs in HAIP; so ISO/IEC TS 18013-7 2nd edition has a chance to point to HAIP for the URI scheme version of OID4VP.

Reply:
good question. I added the following question as a question to ISO:

> Will 18013-7 continue to define how to do mdocs over OID4VP without Digital Credentials API? Or should it be defined in HAIP and 18013-7 annex B can point to HAIP? Thank would resolve the fact that 18013-7 ver 1 Annex B points to OID4VP ID-2, which has diverged from the most recent version of OID4VP?

Reply:
#132 for now